compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Dialog.desktop.kt

```diff
@@ -63,6 +63,13 @@ fun Dialog(
     content = content
 )
 
+@Deprecated(
+    level = DeprecationLevel.HIDDEN,
+    message = "Replaced by DialogWindow",
+    replaceWith = ReplaceWith("DialogWindow(onCloseRequest, state, visible, title, " +
+        "icon, undecorated, transparent, resizable, enabled, focusable, false, " +
+        "onPreviewKeyEvent, onKeyEvent, content)")
+)
```

Comment:
`replaceWith` doesn't make sense for `DeprecationLevel.HIDDEN`. As well as mentioning in release notes.
In case if it requires manual migration it should be deprecated with warning

Reply:
Removed `replaceWith`. Not sure of the harm of mentioning it in the release notes.

Reply:
`DeprecationLevel.HIDDEN` should be transparent for users => no "migration notes"
сс @igordmn

Reply:
Why should it be transparent? There used to be a method they could use (maybe already deprecated) and now they can't (except from already compiled code).

Reply:
The only case when they notice it is when they didn't use named arguments. Probably it is not worth to mention it.

The compiler/IDE just picks the another overload automatically if the used overload is DEPRECATED(HIDDEN)

Reply:
I don't have a strong opinion here, so it is up to you, the author

Reply:
> But in general there's no rule that level=HIDDEN means no release note.

`level=HIDDEN` is used where we have souce compatiable (by existing overloads) and it's required ONLY for binary compatiable. For cases like "named arguments" the reasoning is not so strong, but then it's questionable to use `level=HIDDEN` at the first place

Reply:
I removed the release note.